x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/ml_inference_logic.ts

```diff
@@ -509,30 +509,28 @@ export const MLInferenceLogic = kea<
       },
     ],
     isLoading: [
-      () => [selectors.mlModelsStatus, selectors.mappingStatus],
-      (mlModelsStatus, mappingStatus) =>
-        !API_REQUEST_COMPLETE_STATUSES.includes(mlModelsStatus) ||
-        !API_REQUEST_COMPLETE_STATUSES.includes(mappingStatus),
+      () => [selectors.mappingStatus],
+      (mappingStatus: Status) => !API_REQUEST_COMPLETE_STATUSES.includes(mappingStatus),
```

Comment:
The `isLoading` flag for the pipeline configuration panel no longer depends on the status of model fetching. The "models loading" state is now displayed in the [model selector](https://github.com/elastic/kibana/pull/176257/files#diff-83c7e40124c2eeb3cfbd4081cb93dc8cca1afab4f7ffa53e4d2136e35b8edf10R156-R158). Other steps of the workflow (e.g. generating pipeline configuration) depend on the models data, but it's very unlikely the user will get to that stage before the models finish loading (and even if they do, they just need to wait a little).

If we leave this dependency, the pipeline configuration panel's rendering will be blocked by the fetching of the models.